com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/AuthorizationFilter.java

```diff
@@ -18,6 +19,7 @@
 import com.sap.cloud.lm.sl.cf.web.message.Messages;
 import com.sap.cloud.lm.sl.cf.web.util.SecurityContextUtil;
 
+@Named("authFilter")
```

Comment:
why not `@Named("authorizationFilter")`?

Reply:
Reused the names first but updated the name now